src/features/hotspots/setup/HotspotSetupSelectionScreen.tsx

```diff
@@ -31,26 +35,54 @@ const HotspotSetupSelectionScreen = () => {
   const edges = useMemo((): Edge[] => ['top', 'left', 'right'], [])
   const radii = useBorderRadii()
   const [searchTerm, setSearchTerm] = useState('')
+  const { showOKCancelAlert } = useAlert()
 
   // clear any existing onboarding state
   useEffect(() => {
     dispatch(hotspotOnboardingSlice.actions.reset())
   }, [dispatch])
 
   const handlePress = useCallback(
-    (hotspotType: HotspotType) => () => {
+    (hotspotType: HotspotType) => async () => {
+      const makerHotspot = HotspotMakerModels[hotspotType]
+      const makerArr = Object.keys(Makers).map((makerName) => ({
```

Comment:
We could make a helper function in the `makers/index.ts`? Maybe something like `getMakersList` and could also add a `getMakerDetails`?

Reply:
good call 👍